scripts/source-maintenance.sh

```diff
@@ -40,7 +72,48 @@ fi
 COPYRIGHT_YEARS=`date +"1996-%Y"`
 echo "s/1996-2[0-9]+ The Squid Software Foundation and contributors/${COPYRIGHT_YEARS} The Squid Software Foundation and contributors/g" >>boilerplate_fix.sed
 
-srcformat ()
+# executes the specified command
+# in KeepGoing mode, remembers errors and hides them from callers
+run_()
```

Comment:
Some shells require a space before these brackets.
```suggestion
run_ ()
```

The underscore may also cause problems as it is a reserved character.

Reply:
I applied the suggestion (9a6ad2d3d2dda82a4bb5cce1c1b72317d4ee60d5).

FWIW, I did not find a confirmation regarding underscore being prohibited in function names (POSIX [allows](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_09_05) [it](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap06.html#tag_06_01)). I added that underscore to emphasize that this is an internal helper function and to minimize chances of clashes with programs and built-ins without using long names. Please feel free to rename, of course.